### DISCUSSION TITLE
Trouble installing package TwoSampleMR

### DISCUSSION THREAD

**Discussion by marieklev** [MRCIEU/TwoSampleMR#378] [Q&A] [answered]
Hi! I cant install the TwoSampleMR package, recieving the following errormessage:
ERROR: dependencies ‘car’, ‘MendelianRandomization’, ‘meta’ are not available for package ‘TwoSampleMR’
I have tried to install the different packages manually, but it seems like there is some issues with the different versions?

**Comment by remlapmot** [comment, marked as answer]
Please could you check what version of R you have installed. 

This is because the **meta** package requires a version of R greater than or equal to 4.0.0. R version 4.0.0 was released in April 2020, but I know that some users are still using earlier versions of R in Conda environments etc.

You shouldn't have any problems if you install the latest version of R (version 4.2.1).

**Reply by marieklev** [reply]
Hi! Thanks for answering so quickly, I have R version 4.1.2, so it should be ok?

**Reply by remlapmot** [reply]
In theory that should be fine.

We run a weekly check on the package on several different versions on R. This morning those checks ran successfully for R versions 4.1.3 and 4.0.5 (and more). (You can see them [here](https://github.com/MRCIEU/TwoSampleMR/actions/runs/2902112031))

However, in general it is preferable to use the latest version of R, especially if you are on a Mac or Windows computer. This is because by default R on those platforms installs binary packages from CRAN. CRAN builds those binary packages under the latest version of R and it's preferable, although not essential, to have the packages built under the same version of R that you are using.
